lib/Stats.js

```diff
@@ -405,16 +417,16 @@ Stats.jsonToString = function jsonToString(obj, useColors) {
 			colSizes[col] = 3;
 		for(row = 0; row < rows; row++) {
 			for(col = 0; col < cols; col++) {
-				value = array[row][col] + "";
+				value = getValue(array, row, col) + "";
 				if(value.length > colSizes[col]) {
 					colSizes[col] = value.length;
 				}
 			}
 		}
 		for(row = 0; row < rows; row++) {
 			for(col = 0; col < cols; col++) {
-				var format = row === 0 ? colors.bold : formats[col];
-				value = array[row][col] + "";
+				var format = row === 0 ? colors.bold : array[row][col].color;
```

Comment:
Done, I instead just added `colors.bold` to the header for that table, and removed that conditional.